tensorboard/components/tf_globals/globals-polymer.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2020

Reply:
but... but... commit was made in 2019! Will fix it.